entry/src/main/java/com/github/premnirmal/textcounterdemo/ParanormalSlice.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2020-21 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.github.premnirmal.textcounterdemo;
+
+import ohos.aafwk.ability.AbilitySlice;
+import ohos.aafwk.content.Intent;
+import com.github.premnirmal.textcounter.CounterView;
+import com.github.premnirmal.textcounter.Formatter;
+import java.text.NumberFormat;
+import java.util.Locale;
+
+/**
```

Comment:
This java doc can be removed as sample app content is developed differently for HarmonyOS.

Reply:
Done